src/test/java/com/exasol/adapter/document/documentfetcher/files/BucketFsFileContentTest.java

```diff
@@ -49,17 +55,17 @@ void getInputStreamFailsForMissingFile() {
         final BucketFsFileContent content = create(file);
         final IllegalStateException exception = assertThrows(IllegalStateException.class,
                 () -> content.getInputStream());
-        assertThat(exception.getMessage(), equalTo("F-BFSVS-6: Could not open " + file
-                + ". This is an internal error that should not happen. Please report it by opening a GitHub issue."));
+        assertThat(exception.getMessage(), equalTo("F-BFSVS-6: Could not open '" + file
+                + "'. This is an internal error that should not happen. Please report it by opening a GitHub issue."));
     }
 
     @Test
     void loadAsyncFailsForMissingFile() {
         final Path file = this.tempDir.resolve("non-existing-file");
         final Future<byte[]> content = create(file).loadAsync();
         final ExecutionException exception = assertThrows(ExecutionException.class, () -> content.get());
-        assertThat(exception.getMessage(), equalTo("java.lang.IllegalStateException: F-BFSVS-6: Could not open " + file
-                + ". This is an internal error that should not happen. Please report it by opening a GitHub issue."));
+        assertThat(exception.getMessage(), equalTo("java.lang.IllegalStateException: F-BFSVS-6: Could not open '" + file
```

Comment:
Maybe we can also Rename error codes from BFSVS to VSBFS in this occasion? 
see issue #23

Reply:
✅